### ISSUE TITLE
Graph sensor data from mongodb on helgrindr

### ISSUE THREAD

**Issue by davclark** [glass-bead-labs/data-reflector#4]
@KishPatel1996 it seems like you're currently out in front on this, but cc anyone else that should be involved. I will take the lead on supporting this side of things (@postfalk will focus on the sensor stack).

**Comment by davclark** [comment]
After yesterday, this is working, right?

Before closing, we should document how to set things up - either in the wiki (and I'm not sure which one is the right one...) or for setup, [file issues in the glass-bead-ansible repo](https://github.com/glass-bead-labs/glass-bead-ansible/issues).

**Comment by KishPatel1996** [comment]
What should I write on the wikipedia?  What program I am using and such?

**Comment by davclark** [comment]
Exactly. Have a look at tutorials that you've found useful on the web for inspiration! Be as explicit as you can.

Also - high level stuff can be expressed per this issue: https://github.com/glass-bead-labs/sensor-group/issues/49
